### ISSUE TITLE
Inconsistant use of tabs vs spaces

### ISSUE THREAD

**Issue by squeesh** [andrewt3000/quickPython#2]
In some areas you use tabs for indentation, in other areas you use 4 spaces. 

As per pep8: 
https://www.python.org/dev/peps/pep-0008/#tabs-or-spaces

Specifically:
Python 2 code indented with a mixture of tabs and spaces should be converted to using spaces exclusively.

**Comment by the-zebulan** [comment]
You are right, I noticed that as well. I fixed the spacing (and other PEP8 issues) and added some more examples. 

I submitted a pull request with the changes. Hopefully @andrewt3000 approves of my contributions!
